app/controllers/user_controller.rb

```diff
@@ -411,8 +411,8 @@ def make_friend
     if @new_friend
       if request.post?
         friend = Friend.new
-        friend.user_id = current_user.id
-        friend.friend_user_id = @new_friend.id
+        friend.befriender = current_user
+        friend.befriendee = @new_friend
```

Comment:
Imo here the old names were easier to understand.

Reply:
Yep, those aren't ideal but they are already in use elsewhere. I'm open to suggestions for changing them. I also intend to rename the `Friend` model to `Friendship`, which will make things clearer. But none of that is in this PR.